### DISCUSSION TITLE
Is FastHGT down?

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#7253] [Q&A]
https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.HGTConv.html#torch_geometric.nn.conv.HGTConv this hints that there is a fasthgt but there isn't?

**Comment by wsad1** [comment]
Apologies, FastHGTConv was merged into HGTConv. Will update docs now.
